invenio_db/core.py

```diff
@@ -59,6 +61,9 @@ def init_db(self, app, entrypoint_name='invenio_db.models', **kwargs):
             'sqlite:///' + os.path.join(app.instance_path, app.name + '.db')
         )
         app.config.setdefault('SQLALCHEMY_ECHO', app.debug)
+        app.config.setdefault('DB_VERSIONING_USER_MODEL', None)
```

Comment:
We might have troubles with the initialization order. The only way how to resolve it, which I can think of right now, is to use entry point.

```python
'invenio_db.versioned': [
    'user_cls = invenio_accounts.models:User',
],
```

You can try to load this entry point if there is nothing in config 'DB_VERSIONING_USER_MODEL'